shared/menubar/remote-proxy.desktop.tsx

```diff
@@ -45,7 +46,7 @@ const getIcons = (iconType: BadgeType, isBadged: boolean) => {
   const size = isWindows ? 16 : 22
   const icon = `icon-${platform}keybase-menubar-${badged}${iconType}-${color}-${size}${devMode}@2x.png`
   // Only used on Darwin
-  const iconSelected = `icon-${platform}keybase-menubar-${badged}${iconType}-${colorSelected}-${size}${devMode}@2x.png`
```

Comment:
why remove the badging?

Reply:
removing badging when selected because it looks weird when you select in light mode and the badge moves from being the icon being colorful to the orange dot; IMO it also looks a little odd for the orange dot to stay when it's selected in dark mode, but that's a thing I feel less strongly about.

Reply:
@adamjspooner / @cecileboucheron you ok with this

Reply:
taking this line out into a separate PR